sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -231,7 +231,7 @@ private[hive] class HiveClientImpl(
     ugi.getShortUserName
   } catch {
     case e: Exception =>
-      logWarning("Can not get login user.")
+      logError("Can not get login user.")
```

Comment:
Same comment here: I'm not sure we really want to log error message here, as I expect thrown exception will make query fail somewhere in caller side, and it will log the exception correctly - stack trace in exception will tell us.